marilib/marilib.py

```diff
@@ -35,6 +35,7 @@ class MariLib:
     test_rate: int = 0
     test_load: int = 0
```

Comment:
I think all parameters that begin with `test_` should be in another class that just handles the test case. 

I think it is fine to keep a single reference to `latency_tester` in MariLib (as you already have it). But let's avoid spreading lots of code about test, in regular classes that are not related to measurements. In other words, just focus on separation of concerns. 

To give more context, MariLib has to be generic to work well in the Edge mode, and soon we will need to expand it to also work in the Cloud mode. If we have the (latency) test code cleanly separated, it will be easy to refactor. In general, always better to strive for [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns).